models/migrations/v1_18/v230.go

```diff
@@ -7,11 +7,16 @@ import (
 	"xorm.io/xorm"
 )
 
+type oAuth2Application struct {
+	ID                 int64
+	ConfidentialClient bool `xorm:"NOT NULL DEFAULT TRUE"`
+}
+
+func (oAuth2Application) TableName() string {
+	return "oauth2_application"
+}
+
 // AddConfidentialColumnToOAuth2ApplicationTable: add ConfidentialClient column, setting existing rows to true
 func AddConfidentialClientColumnToOAuth2ApplicationTable(x *xorm.Engine) error {
-	type OAuth2Application struct {
-		ConfidentialClient bool `xorm:"NOT NULL DEFAULT TRUE"`
-	}
-
-	return x.Sync(new(OAuth2Application))
+	return x.Sync(new(oAuth2Application))
```

Comment:
Does this work?

```
func AddConfidentialClientColumnToOAuth2ApplicationTable(x *xorm.Engine) error {
  type oauth2Application struct {
	  ID                 int64
	  ConfidentialClient bool `xorm:"NOT NULL DEFAULT TRUE"`
  }

  return x.Sync(new(oauth2Application))
}
```

Reply:
c41c6d2515c78e52e79d677e85fefb2893cd2393